### ISSUE TITLE
Hangs when exporting massive issue

### ISSUE THREAD

**Issue by mfbx9da4** [excalidraw/excalidraw#4678]
[Example canvas](https://gist.github.com/mfbx9da4/45c83e655e1af524e5c7dd3fd7043ff3)

At the very least some feedback to the user that it is still trying to export would be good!

**Comment by dwelle** [comment]
Yeah, we're aware of this. Apart from rendering in a worker ([https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas](OffscreenCanvas) is sadly not support on FF and Safari), we'd have to split the image into tiles and glue the bitmaps together.
